tests/foreman/ui/test_contenthost.py

```diff
@@ -161,6 +160,7 @@ def test_positive_end_to_end(session, repos_collection, vm):
     result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
     with session:
+        session.location.select('Default Location')
```

Comment:
Instead of being to direct with `Default Location` constant, access `default_location` fixture and then use to it to select the location by its name.

```suggestion
        session.location.select(default_location.name)
```

Refer a quick idea PR raised by me for the high pipeline failure analysis, I have done : https://github.com/SatelliteQE/robottelo/pull/9195

Reply:
Else you can point to the Constant https://github.com/SatelliteQE/robottelo/blob/master/robottelo/constants/__init__.py#L638 but you have to additionally import the constants, so the above approach is better always !

Reply:
@jyejare Its because I wanted to avoid importing things that I used the method I did. If I use a fixture than it searches for the default location, I cannot see how that is helpful, seems this is just a mater of style. If I make the change you suggest then I lose @JacobCallahan approval, so one step forward one step back. I will ask in reviewer's meeting for other opinions.

Reply:
@swadeley No, u need not import anything, the fixture is easily available since it's a global fixture. Also, we can avoid future failures in case the name of `default location` is changed to something else.

Reply:
OK, will do